bindings/node/lib/autoEncrypter.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+const mc = require('bindings')('mongocrypt');
+const common = require('./common');
+const databaseNamespace = common.databaseNamespace;
+const StateMachine = require('./stateMachine').StateMachine;
+
+/**
```

Comment:
My goal in keeping this all in the module was so that we could keep documentation isolated, similar to kerberos. I think we should speak more with docs about this, but I think for now it's early days and we can decide to move it by the 4.2 release if need be.

Reply:
We could still have it be part of the documentation, similar to how we handle `bson` [here](https://github.com/mongodb/node-mongodb-native/blob/fb0718689ca74deeb2a3471af199257758babd11/conf.json#L27)

Reply:
agreed